src/text/writer.rs

```diff
@@ -364,9 +410,11 @@ mod tests {
     }
 
     #[test]
-    #[ignore]
     fn write_clob() {
-        todo!();
+        writer_test(
+            |w| w.write_clob(&[b'a', b'"', b'\'', b'\n']),
```

Comment:
For an easier way to get a byte array representing text, I suggest making a `&str` and then calling [`as_bytes()`](https://doc.rust-lang.org/std/primitive.str.html#method.as_bytes) on it:
```rust
"Algeria".as_bytes()
```
You can also use `r""` syntax to create a [raw string literal](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals), allowing you to write characters you would otherwise have to escape.